### PR TITLE
CLDC-3123 Speed up deployment pipeline

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**

### CODE REVIEW

File: .github/workflows/aws_deploy.yml [hunk 1]
```diff
@@ -59,13 +59,17 @@ jobs:
...
 
-      - name: Build, tag, and push docker image to ECR
+      - name: Build, tag, and push docker image to ECR if there is no image, failing for releases
         id: build-image
         if: ${{ env.image-exists == 'false' }}
         env:
           registry: ${{ steps.ecr-login.outputs.registry }}
           commit_tag: ${{ github.sha }}
         run: |
+          if [[ ${{ inputs.environment }} == 'production' ]]; then
```
Comment (RobertSullivanSoftwire) [inline .github/workflows/aws_deploy.yml:69]: This mostly looks good to me - just wanted to check we're not deleting images from the registry (e.g. due to age), first.

**Review by RobertSullivanSoftwire — APPROVED** [review]

File: .github/workflows/aws_deploy.yml (hunk 1 above)
Comment (natdeanlewissoftwire) [reply .github/workflows/aws_deploy.yml:69]: Yep I'll look into this, good point 👍

**Review by RachaelBooth — APPROVED** [review]
LGTM

File: .github/workflows/aws_deploy.yml (hunk 1 above)
Comment (RachaelBooth) [reply .github/workflows/aws_deploy.yml:69]: N.B. There aren't currently deletion rules, although there are plans for some: https://dluhcdigital.atlassian.net/browse/CLDC-2600 - have commented there on the need to take this into account.

Comment (natdeanlewissoftwire) [reply .github/workflows/aws_deploy.yml:69]: Thanks!

**Pull request merged**